views/order/edit.php

```diff
@@ -0,0 +1,45 @@
+<section class="container-fluid">
+    <div class="card my-4 mt-5">
+        <div class="card-header p-0 position-relative mt-n4 mx-3 z-index-2">
+            <div class="d-flex justify-content-between bg-gradient-primary shadow-primary border-radius-lg pt-4 pb-3">
+                <h6 class="text-white text-capitalize ps-3 mt-1">Editar Pedido</h6>
+
+                <a href="/pedidos/listar" class="btn btn-light mb-1 me-4">
+                    Listar
+                </a>
+            </div>
+        </div>
+        <div class="card-body p-5">
+            <form method="post" action="">
+                <div class="input-group input-group-outline mb-3">
+                    <label class="w-30" for="type">
+                        Tipo
+                        <input type="text" name="type" id="type" class="form-control">
+                    </label>
+                </div>
+
+                <div class="input-group input-group-outline mb-3">
+                    <label class="w-30" for="items">
+                        Items
+                        <input type="text" name="items" id="items" class="form-control">
+                    </label>
+                </div>
```

Comment:
- [ ] Sugestão: Para o input dos itens, seria interessante utilizar Bootstrap Tags Input ou um Bootstrap Textarea. 
Para uma abordagem mais completa e complexa: 
1. Utilizar um datalist para escolher o item desejado.
2. Após isso, o label do item selecionado aparece embaixo do datalist
3. e surge um input ao lado desse label onde é possível informar a quantidade.

Justificativa: Caso seja apenas um input normal, pode ser que conforme for necessário adicionar itens, a visualização seja prejudicada. Ex: "ItemA x1, ItemB x5, ItemC, x6, ItemD x1" ou "Item A 1, Item B 2, Item C 5, Item D 6". Lembrando que é apenas uma sugestão.